packages/app/components/Header/Header.tsx

```diff
@@ -0,0 +1,47 @@
+"use client";
+import Image from "next/image";
+import Logo from "../../public/assets/images/logo.svg";
+import Stackly from "../../public/assets/images/stackly.svg";
+import Blocks from "../../public/assets/images/blocks.svg";
+
+import Link from "next/link";
+import MobileMenu from "./MobileMenu";
+
+export default function Header() {
+  return (
+    <header className="h-[72px] sticky top-0 z-50 flex flex-col w-full bg-surface-25 border-b border-solid border-b-surface-75">
+      <nav className="h-full flex items-center w-full max-w-screen-xl px-4 mx-auto">
+        <div>
+          <Link href="/" title="Stackly Home" className="flex items-center">
+            <Image aria-label="Stackly logo" src={Logo} alt="Stackly Logo" />
+            <Image
+              aria-label="Stackly"
+              src={Stackly}
+              alt="Stackly Logo"
+              className="ml-[12px] hidden md:block"
+            />
+          </Link>
+        </div>
+        <div className="border-r border-solid border-b-gray-100 h-[31px] ml-[20px] md:ml-[40px]"></div>
```

Comment:
**We should avoid using custom sizes! Only use it when is straighly necessary.**

For eg. we have the `h-8` which is `32px` I don't think that `1px` will matter in this case.

By using the tailwind scale we benefit the consistency.

```suggestion
        <div className="border-r border-solid border-b-gray-100 h-8 ml-5 md:ml-10"></div>
```